### DISCUSSION TITLE
Add folder to bentoml boundle

### DISCUSSION THREAD

**Discussion by zfallahnejad** [bentoml/BentoML#2095] [Q&A] [answered]
Hello everyone
Thanks for your great work. I want to ask how can I add a folder to the BentoML bundle? Is it possible to add a folder as an artifact or copy a folder into a bundle using `setup_sh`?

**Comment by parano** [comment, marked as answer]
Hi @zfallahnejad yes it is possible, in BentoML 0.13.*, we recommend a workaround to directly copy files to the bundle path after the `save` call. e.g.:

```
saved_path = svc.save()
# copy files or folders to {saved_path}
shutils.copy(..., saved_path)
```

In the upcoming 1.0 version, copying files and directory to Bento bundle will be supported out of the box.
